examples/main.rs

```diff
@@ -11,12 +11,12 @@ enum Tile {
     Grass,
 }
 
-impl Tile {
-    fn to_char(&self) -> char {
-        match self {
-            Tile::Dirt => '🟫',
-            Tile::Grass => '🟩',
-            Tile::Water => '🟦',
+impl From<Tile> for String {
```

Comment:
impl Into<String> for Tile would be more idiomatic...

Reply:
Doing

```rs
impl Into<String> for Tile {
    fn into(self) -> String {
        match self {
            Tile::Dirt => "\x1b[1;30m██\x1b[0m".to_string(),
            Tile::Grass => "\x1b[1;32m██\x1b[0m".to_string(),
            Tile::Water => "\x1b[1;34m██\x1b[0m".to_string(),
        }
    }
}
```

Gives me a clippy warning:

```
an implementation of `From` is preferred since it gives you `Into<_>` for free where the reverse isn't true
`#[warn(clippy::from_over_into)]` on by default
consider to implement `From<Tile>` instead
for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#from_over_into
```

Not sure what's best here

Reply:
Welp, use From<Tile> then.